src/composables/queries/useUserPoolsQuery.ts

```diff
@@ -77,11 +77,7 @@ export default function useUserPoolsQuery(
     });
 
     for (let i = 0; i < pools.length; i++) {
-      const isComposableStableLikePool = isComposableStableLike(
-        pools[i].poolType
-      );
-
-      if (isComposableStableLikePool) {
+      if (isDeep(pools[i])) {
```

Comment:
You were right actually, this needs to be a check if it's ComposableStable like.
```suggestion
      if (isComposableStableLike(pools[i].poolType)) {
```

Reply:
fixed here and below similar condition